sdcm/utils/alternator/api.py

```diff
@@ -30,17 +31,31 @@ def __init__(self, sct_params):
         self.alternator_apis = {}
 
     def create_endpoint_url(self, node):
-        return 'http://{}:{}'.format(normalize_ipv6_url(node.external_address), self.params.get("alternator_port"))
+        if node.is_kubernetes():
+            web_protocol = "http" + ("s" if self.params.get("alternator_port") == 8043 else "")
```

Comment:
this `8043` looks like magic number... 
can't we switch to a config `alternator_use_https` and set proper port then?